test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,445 @@
+const { strict: assert } = require('assert');
+const {
+  convertToHexValue,
+  withFixtures,
+  getWindowHandles,
+} = require('../helpers');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Send token from inside MetaMask', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('starts to send a transaction, transitions to the confirm screen, displays the token transfer data, customizes gas, submits the transaction, finds the transaction in the transactions list', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'connected-state',
```

Comment:
there is a fixture for a `custom-token`, could we use it instead of this one? This might help reduce the test steps, as we won't need to add the token to the wallet then

Reply:
Good idea. I tried it but when I run test it shows me 0 TST so I'm not able to send tokens since I don't have enough tokens. Maybe there is way to change custom-token fixture in order to have more TST tokens?